### PR TITLE
Add release artifacts validation script pt 164978469

### PR DESCRIPTION
**Pull request by valerifilipov**
In scope of [PT-164978469](https://www.pivotaltracker.com/story/show/164978469)
Successful run https://circleci.com/gh/aeternity/aeternity/22338
Failed run https://circleci.com/gh/aeternity/aeternity/22375

Requires aeternity/infrastructure#349 to be fixed and merged to stable

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -833,6 +833,17 @@ jobs:
...
+
+
+  verify_release_artifacts:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    steps:
+      - run:
+          name: Verify release artifacts
+          command: |
+            /infrastructure/scripts/check_release_artifacts.sh ${CIRCLE_TAG}
```
Comment (lucafavatella) [inline .circleci/config.yml]: Depends on https://github.com/aeternity/infrastructure/pull/349/files
in `stable` branch.

Comment (lucafavatella) [reply .circleci/config.yml]: I see it is in description, thanks.

Comment (dincho) [inline .circleci/config.yml]: could you please change this to relative path ?

Comment (valerifilipov) [reply .circleci/config.yml]: done in 1f61b41

**Comment by lucafavatella** [comment]
> Requires [aeternity/infrastructure#349](https://github.com/aeternity/infrastructure/pull/349) to be fixed and merged to stable

I see included in stable as https://github.com/aeternity/infrastructure/commit/4539fd4ac6a02fc95b5db140e36c231d41678c34

File: .circleci/config.yml [hunk 2]
```diff
@@ -834,6 +834,18 @@ jobs:
...
+
+  verify_release_artifacts:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    working_directory: /infrastructure
+    steps:
+      - run:
+          name: Verify release artifacts
+          command: |
+            ./scripts/check_release_artifacts.sh ${CIRCLE_TAG}
```
Comment (lucafavatella) [inline .circleci/config.yml]: I see https://github.com/aeternity/infrastructure/commit/4539fd4ac6a02fc95b5db140e36c231d41678c34

Comment (lucafavatella) [reply .circleci/config.yml]: I see Windows package not checked. And it makes sense, as upload still manual.

**Pull request merged**